src/routes/auth.js

```diff
@@ -2,16 +2,18 @@ import { Router } from 'express';
 import passport from 'passport';
 
 import authController from '../controllers/auth';
+import Authentication from '../middlewares/authentication';
```

Comment:
I wonder why this starts with a capital letter when it isn't a class. Please adjust it

Reply:
Nice catch. My bad. I originally implemented it as a class. I have made the correction now

Reply:
My bad. Nice catch. I originally implemented that as a class but failed to make that correction after refactoring the code.